nf_core/subworkflows/lint/main_nf.py

```diff
@@ -80,6 +91,16 @@ def main_nf(_, subworkflow: NFCoreComponent) -> Tuple[List[str], List[str]]:
     else:
         subworkflow.passed.append(("main_nf_script_outputs", "Workflow 'emit' block found", subworkflow.main_nf))
 
+    # Check for only one workflow per file
+    if subworkflow_nb > 1:
+        subworkflow.failed.append(
```

Comment:
Let's make it a warning to not block some of the current subworkflows in the modules repo
```suggestion
        subworkflow.warned.append(
```